openedx/openedx-platform
# feat: Tag count in components [FC-0036]

## Description

This adds the tag count button on components:

![image](https://github.com/openedx/edx-platform/assets/11827305/11aa905a-6574-4092-867d-49fbbda3a417)

## Supporting information

- Internal link: [FAL-3573](https://tasks.opencraft.com/browse/FAL-3573)
- Closes https://github.com/openedx/modular-learning/issues/158
- Uses: https://github.com/openedx/openedx-learning/pull/133

## Testing instructions

- Enable `contentstore.enable_copy_paste_units` and `new_studio_mfe.use_tagging_taxonomy_list_page` flags.
- Run the [taxonomy-sample-data](https://github.com/open-craft/taxonomy-sample-data).
- Go to a unit and verify that the tag count button appears on the components.
- Create a new component and verify that the tag count button doesn't appear.
- Add tags using "Manage tags".
- Refresh the page and verify that all the components has the tag count button.
- Disable the `new_studio_mfe.use_tagging_taxonomy_list_page` flag.
- Refresh the page and verify that the tag count buttons are gone.

## Other information

- The count update is missing if the tags are edited in the manage tags drawer.
